### PR TITLE
A small fix.

### PR DESCRIPTION
**Pull request by NeuroDonu**
In my fork, I fixed the get_example function in app.py because it wasn't working due to images not working. In face_adapter_pipline.py, the code on lines 44-47 was put under with torch.no_grad():, which avoided the following errors: 
this
If we try to follow the instructions to replace with tensor.detach().numpy() we will get a tensor error. so my fix was the only working solution I had :D

### CODE REVIEW

**Comment by FaceAdapter** [comment]
Thank you :) Have merged~

**Comment by FaceAdapter** [comment]
Our runnable scripts for gradio demo here~ https://huggingface.co/spaces/FaceAdapter/FaceAdapter/blob/main/app.py
